### PR TITLE
added swagger documentation for ResourceController IdentityController…

### PR DESCRIPTION
**Pull request by Eldiar004**
… and changed return type to ResponseEntity for Resource and S3

### CODE REVIEW

File: src/main/java/com/levante/levante/web/controller/identity/IdentityController.java [hunk 1]
```diff
@@ -18,39 +22,87 @@
...
+    })
     public ResponseEntity<UserDetailsDTO> registerUser(@RequestBody UserDetailsDTO details) {
         return new ResponseEntity<>(identityService.registerUser(details), HttpStatus.CREATED);
     }
 
+    // ---------------------------------------
     @GetMapping("/identity/users")
+    // ---------------------------------------
+    @Operation(
+            summary = "Get All User",
```
Comment (i-Coderr) [inline src/main/java/com/levante/levante/web/controller/identity/IdentityController.java:48]: rewrite to Get all users

File: src/main/java/com/levante/levante/web/controller/identity/IdentityController.java [hunk 2]
```diff
@@ -18,39 +22,87 @@
...
             Pageable pageable) {
         return ResponseEntity.ok(identityService.getAllUsers(filter, langId, pageable));
     }
 
+    // ---------------------------------------
     @PutMapping("/identity/users/verify")
+    // ---------------------------------------
+
+    @Operation(
+            summary = "Verification user",
```
Comment (i-Coderr) [inline src/main/java/com/levante/levante/web/controller/identity/IdentityController.java:66]: rewrite to User verification

File: src/main/java/com/levante/levante/web/controller/identity/IdentityController.java [hunk 3]
```diff
@@ -18,39 +22,87 @@
...
         identityService.verifyUser(verificationCode);
         return ResponseEntity.ok("Success");
     }
 
+    // ---------------------------------------
     @PostMapping("/identity/users/verification")
+    // ---------------------------------------
+
+    @Operation(
+            summary = "Send Verification user",
```
Comment (i-Coderr) [inline src/main/java/com/levante/levante/web/controller/identity/IdentityController.java:83]: Send verification to the user

**Review by i-Coderr — COMMENTED** [review]
@Eldiar004 comments

File: src/main/java/com/levante/levante/web/controller/identity/IdentityController.java (hunk 3 above)
Comment (Eldiar004) [reply src/main/java/com/levante/levante/web/controller/identity/IdentityController.java]: done

File: src/main/java/com/levante/levante/web/controller/identity/IdentityController.java (hunk 2 above)
Comment (Eldiar004) [reply src/main/java/com/levante/levante/web/controller/identity/IdentityController.java]: done

**Review by i-Coderr — APPROVED** [review]
Approved

**Pull request merged**